### PR TITLE
Release PREP, media and metadata

### PR DESCRIPTION
**Pull request by SzymonSt**

### CODE REVIEW

File: .DS_Store
Comment (MRadzynski) [inline .DS_Store:1]: Is `.DS_Store` file needed? It's probably an internal MacOS file

File: signal0ne/.DS_Store
Comment (MRadzynski) [inline signal0ne/.DS_Store:1]: same as above

File: signal0ne/README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
\ No newline at end of file
+# Signal0ne
+Signal0ne is an ultimate AI debugging and issue resolution tool.
+See the demo at: https://signaloneai.com/#pro
+
+#### Features
+
+- Issue analysis
+
+- Code fix tip(PRO only)
```
Comment (MRadzynski) [inline signal0ne/README.md]: `Code fix tips (PRO only)`

File: signal0ne/README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+# How to use it
+
+#### Download docker desktop extension to run agent that will discover issues.
```
Comment (MRadzynski) [inline signal0ne/README.md]: + `from your docker containers`

File: signal0ne/README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+
+#### Subscribe to PRO version(it is free for a month without a need to provide payment details)
+
+![PRO](https://github.com/Signal0ne/signal0ne-vscode-extension/blob/c3c776d00236c97c7704b8082e0dcae8bb18cd34/signal0ne/resources/docker_desktop_pro.png?raw=true)
+
+#### Login to both extensions and let it run in the background to discover all issues in your running applications you are developing
+
+![Issues View](https://github.com/Signal0ne/signal0ne-vscode-extension/blob/3fcee544f0ec251166c49e483133a1c73c64ae10/signal0ne/resources/vscode_issues_view.png?raw=true)
+
+#### Add code to context with "Code fix" feature to get proposed solution of issue in your code
```
Comment (MRadzynski) [inline signal0ne/README.md]: maybe rewrite it to something like this: 
[Just a suggestion] `Select the issue from the list and select the code block that you think can be related to the issue and select the 'Fix code' option from the context menu to try fix the issue using our extension`

**Review by MRadzynski — CHANGES_REQUESTED** [review]
Great job! Just a few suggestion

File: .DS_Store
Comment (SzymonSt) [reply .DS_Store:1]: wtf I don't have such file in the directory

Comment (MRadzynski) [reply .DS_Store:1]: its hidden by default I guess

**Pull request merged**